### PR TITLE
Added ques to C++

### PR DESCRIPTION
**Pull request by ishubham010**
Hello Sir, I have added a Question to C++. Please accept my pull request.

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -11050,6 +11050,16 @@
...
+    {
+      "login": "ishubham010",
+      "name": "Shubham Choubey",
+      "avatar_url": "https://avatars.githubusercontent.com/u/71643172?v=4",
+      "profile": "https://github.com/ishubham010",
+      "contributions": [
+        "design",
+        "code"
+      ]
+    },
```
Comment (Ebazhanov) [inline .all-contributorsrc:11062]: do not rush, this is automacticaly generated by all-contributors plugin, so please remove it. It can be added later 💯

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
@ishubham010 left comment

File: .all-contributorsrc (hunk 1 above)
Comment (ishubham010) [reply .all-contributorsrc]: I apologies for the mistake, Finally submitting clean code hope this time it will get commited.

**Review by Ebazhanov — APPROVED** [review]
Looks good now, thank you @ishubham010 for your contribution 👍

**Comment by Ebazhanov** [comment]
@all-contributors please add @ishubham010 for design, code

**Pull request merged**